client-app/ui-kit/components/atoms/date-selector/vc-date-selector.vue

```diff
@@ -4,64 +4,44 @@
     <span v-if="isRequired" class="text-[color:var(--color-danger)]">*</span>
   </div>
   <input
-    class="box-border h-11 w-full min-w-0 appearance-none rounded border border-gray-300 bg-white p-3 text-base leading-none outline-none focus:border-gray-400"
-    type="date"
-    :value="inputValue"
+    v-model="value"
```

Comment:
`v-model` is `:value` + `@input`/`@change`. Use one of it. Write with `:value` or use `composableWriteRef`